Sources/HaCWebsiteLib/ViewModels/PostCard.swift

```diff
@@ -30,3 +31,7 @@ struct PostCard {
     )
   }
 }
+
+protocol PostCardRepresentable {
```

Comment:
@jaredkhan is this what you had in mind?

Reply:
Looks good!